pep-0605.rst

```diff
@@ -0,0 +1,875 @@
+to allow for initial publication of potential stable ABI additions without
+immediately committing to their long term inclusion::
+
+    #define Py_LIMITED_API_PROVISIONAL 0xFFFFFFFF
+
+    #if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= Py_LIMITED_API_PROVISIONAL
+    // A provisional addition to the stable ABI would appear here
+    #endif
+
+(The equivalent Python version for this value would be 255.255.255f16)
+
+This PEP is NOT proposing the immediate introduction of such a marker, as
+"publish an alpha release instead of a beta release" seems like a simpler
+way to handle such a situation if it arises. However, it still seemed worth
+noting as a potential future addition in the event that a stable ABI enhancement
+is needed to serve a particular purpose, and publishing an alpha release isn't
+considered an adequate option for obtaining design feedback.
+
+
+Implications for Python library development
+-------------------------------------------
+
+By using the "beta" labelling rather than the incremental feature release
+numbering proposed in PEP 598, the hope would be that consumers of these
+new rolling releases would realise that they're likely going to need to build
+their own wheel archives from source, and will generally be more prone to
+encountering library compatibility issues when updating to a new release.
+
+Library authors who actually want to support the beta stream would have the
+option of testing against the latest beta release in their pre-merge test
+matrices (just as they test against the latest maintenance release of previously
+published versions), with the CPython nightly builds offered by some CI
+providers used solely in an advisory capacity for early detection of potential
+compatibility problems.
+
+Having a rolling beta release stream available may also make it more feasible
+for more CI providers to offer a "CPython beta release" testing option. At the
+moment, this feature is only available from CI providers that are willing and
+able to put the necessary time and effort into creating, testing, and publishing
+their own builds from the CPython master branch (e.g. [6_]).
+
+
+Implications for the proposed Scientific Python ecosystem support period
+------------------------------------------------------------------------
+
+Based on discussions at SciPy 2019, NEP (NumPy Enhancement Proposal) 29 has
+been drafted [3_] to propose a common convention across the Scientific Python
+ecosystem for dropping support for older Python versions.
+
+While the exact formulation of that policy is still being discussed, the initial
+proposal is very simple: support any Python feature release published within
+the last 42 months.
+
+For an 18 month feature release cadence, that works out to always supporting at
+least the two most recent feature releases, and then dropping support for all
+X.Y.z releases around 6 months after X.(Y+2).0 is released. This means there is
```

Comment:
```suggestion
X.Y.Z releases around 6 months after X.(Y+2).0 is released. This means there is
```

Reply:
For Nick's benefit, I posted an alternative suggestion for this part below.

Reply:
Just went with capitalisation change for now, albeit for reasons that are probably to subtle to be worth caring about...